### DISCUSSION TITLE
Text pasting as... an image?

### DISCUSSION THREAD

**Discussion by krangor** [buttondown/discussions#42] [Q&A]
Hi! A couple of times, recently, I've pasted a block of text from a Word doc to a newsletter-in-progress, and found that it arrives as an image, almost like a screenshot of the Word doc, rather than plain (or styled) text. Is this intended behaviour? Something weird Word is doing? Something weird my clipboard manager (Pastebot) is doing?

It works fine if I invoke Pastebot and ask it manually to paste plain text.

**Comment by jmduke** [comment]
Huh, interesting. Any chance you could provide an example?

**Comment by krangor** [comment]
Sent!

—
Mark Slutsky
***@***.***
www.markslutsky.com
—

**Reply by jmduke** [reply]
Huh — definitely seems like it might be a Pastebot thing. We don't do any PNG-ification on our end (though Word's 'rich' format is very janky).

**Comment by krangor** [comment]
Fascinating! Will double-check with Pastebot closed to make extra sure.

—
Mark Slutsky
***@***.***
www.markslutsky.com
—

**Comment by krangor** [comment]
OK, so, I think it’s a Word thing, as it still happened with Pastebot closed, and only seems to be happening with stuff copied from Word.

Some more examples of it in the wild:

https://answers.microsoft.com/en-us/msoffice/forum/all/whenever-i-copy-and-paste-something-from-word-onto/6ee75e6c-5970-4156-aaac-e305cb48031f <https://answers.microsoft.com/en-us/msoffice/forum/all/whenever-i-copy-and-paste-something-from-word-onto/6ee75e6c-5970-4156-aaac-e305cb48031f>

https://wordpress.com/forums/topic/pasted-text-becomes-image/?view=all <https://wordpress.com/forums/topic/pasted-text-becomes-image/?view=all>



—
Mark Slutsky
***@***.***
www.markslutsky.com
—

**Comment by jmduke** [comment]
Wild, huh. Can you try pasting without formatting (Shift + Cmd/Ctrl + V)? I wonder if that gives us the plaintext.

**Comment by jmduke** [comment]
Closing this out due to inactivity! @krangor, feel free to re-open if you try the above.
